app/components/Arena/EventCard.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import React from "react";
+import { useRouter } from "next/navigation";
+
+import Button from "../Button";
+
+const EventCard = ({ title, description, image, links }: any) => {
+  const router = useRouter();
+
+  return (
+    <div
+      className="mb-6 flex w-[90vw] max-w-md cursor-pointer flex-col items-start justify-start transition-transform duration-300 hover:scale-105"
+      onClick={() => router.push(links[0].url)}
+    >
+      <img className="h-48 w-full object-cover" src={image} alt={title} />
+      <div className="flex w-full flex-col items-start justify-start gap-1 rounded-tl-lg bg-stone-800 px-3.5 py-3">
+        <div className="w-full font-['Archivo'] text-base leading-normal font-bold tracking-tight text-white">
+          {title}
+        </div>
+        <div className="w-full font-['Chakra_Petch'] text-sm leading-tight font-normal tracking-tight text-white">
+          {description}
+        </div>
```

Comment:
- Use [semantic elements](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html) like `<h2>` and `<p>`.
- Use fonts defined by variables in `globals.css` like `font-sans`. [Learn more.](https://tailwindcss.com/docs/font-family#customizing-your-theme)
- Use font sizes and line heights defined by variables in `globals.css` like `text-title-md` and `leading-title-md`. You should see what variable to use by going into Dev Mode (`Shift`+`D`) in Figma.